Classes/Projection/ProjectorInterface.php

```diff
@@ -22,6 +22,16 @@
      * Returns the class name of the (main) Read Model of the concrete projector
      *
      * @return string
+     * @api
      */
     public function getReadModelClassName(): string;
+
+    /**
+     * Removes all objects of this repository as if remove() was called for all of them.
+     * For usage in the concrete projector.
+     *
+     * @return void
+     * @api
+     */
+    public function drop();
```

Comment:
discussed this in length Karsten some time ago, we ended up with "drop". We also considered "destroy", "flush", "reset", "empty", "clear".

"reset" could work as well, but I think it is less common for a task which removes data.